symengine/mp_class.h

```diff
@@ -60,6 +60,18 @@ inline integer_class operator"" _z(const char *str)
 {
     return integer_class(str);
 }
+
+inline rational_class operator"" _q(const char *str)
+{
+    return rational_class(integer_class(str));
+}
+}
+
+inline void mp_tdiv_qr(rational_class &q, rational_class &r,
+                       const rational_class &a, const rational_class &b)
+{
+    q = a / b;
+    r = 0;
```

Comment:
You don't have to use this kind of hacks. It's better to write two functions for `divides_upoly`